modules/ts/misc/run.py

```diff
@@ -850,12 +850,49 @@ def getRunArgs(args):
             path = npath
     return run_args
 
+def moveTests(instance, destination):
+    src = os.path.dirname(instance.tests_dir)
+    # new binaries path
+    newBinPath = os.path.join(destination, "bin")
+
+    try:
+        # copy binaries and CMakeCache.txt to the specified destination
+        shutil.copytree(src, newBinPath)
```

Comment:
Some info about RPATH and handling by CMake: http://www.cmake.org/Wiki/CMake_RPATH_handling